yarn-project/end-to-end/src/e2e_inclusion_proofs_contract.test.ts

```diff
@@ -0,0 +1,66 @@
+import { AccountWallet, CompleteAddress, PXE } from '@aztec/aztec.js';
+import { InclusionProofsContract } from '@aztec/noir-contracts/types';
+
+import { jest } from '@jest/globals';
+import { type MemDown, default as memdown } from 'memdown';
+
+import { setup } from './fixtures/utils.js';
+
+export const createMemDown = () => (memdown as any)() as MemDown<any, any>;
+
+const TIMEOUT = 90_000;
+
+describe('e2e_inclusion_proofs_contract', () => {
```

Comment:
Adding a for inclusion in the nullifier tree would also be nice to have.

Reply:
Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3255/commits/c5a10e47fbe034ff7743dcfbd9b31498c229d722 and https://github.com/AztecProtocol/aztec-packages/pull/3255/commits/3c89221a8113524ecfeaff99c4b70e04afddbaaa